src/Blazored.Toast/BlazoredToast.razor.cs

```diff
@@ -42,6 +44,25 @@ protected override async Task OnInitializedAsync()
     public void Close()
         => ToastsContainer.RemoveToast(ToastId);
 
+    private void MouseOver(MouseEventArgs e)
```

Comment:
I think this method name should be more descriptive. Perhaps `AttemptPause` to signify that it might not pause, depending on the setting.

Reply:
What do you think about `TryPauseCountdown` and `TryResumeCountdown`  or we go with Attempt?

One more thing, i am just thinking but i am probably wrong. This is bind to @onmouseover and @onmouseout and setting methods to AttemptPause and AttemptResume is good in this case but what if in feature development we would like to have some animations too when @onmouseover is triggered? I am just learning what are best practices leave it like that and later refactor if there will be need?

Reply:
Those are great names, I much prefer `Try...` over `Attempt...`.

Good question. I'd always choose to name methods based on what they're doing right now. Names can be refactored when their actions change.